test/test/boot.js

```diff
@@ -404,6 +404,119 @@ function configureJasmineEnvironment() {
   };
 }
 
+/**
+ * Patch Cast's cast.__platform__.canDisplayType to allow it to operate across
+ * frames and origins in our testing environment.
+ *
+ * The Cast runtime only exposes cast.__platform__ on the top window, not
+ * iframes embedded within it.  However, both Chromecast WebDriver Server in
+ * our lab and the test runner Karma use iframes, and there are three different
+ * origins involved: WebDriver Server's receiver at github.io, Karma's
+ * top-level, and an inner frame of Karma that is raw HTML as text with no
+ * origin.
+ *
+ * With all of these complexities, the only way to access cast.__platform__ is
+ * asynchronously via postMessage.  This means all callers of
+ * cast.__platform__.canDisplayType must use `await`, even though the
+ * underlying method is synchronous.  If the caller is running in the top frame
+ * (as in a real receiver), `await` will do no harm.  If the caller is running
+ * inside our tests in Karma, the `await` is critical to access __platform__
+ * via this shim.
+ */
+function patchCastCanDisplayType() {
+  if (!navigator.userAgent.includes('CrKey')) {
+    // Not Chromecast.  Do nothing.
```

Comment:
nit; Extra space.

Reply:
Old American typing instruction... a bad habit.  I'll fix it.

I was taught old typewriter style in elementary school.

https://www.cultofpedagogy.com/two-spaces-after-period/